src/Sarif/PerLanguageOption.cs

```diff
@@ -48,7 +48,7 @@ public PerLanguageOption(string feature, string name, Func<T> defaultValue, stri
 
             if (string.IsNullOrWhiteSpace(name))
             {
-                throw new ArgumentException(nameof(name));
+                throw new ArgumentNullException(nameof(name));
```

Comment:
I'm fine with the proposed change and would actually prefer not to over-elaborate on exception handling in this instance (given that we do not enforce a similar bar elsewhere in the code).